via/static/scripts/video_player/hooks/use-app-layout.ts

```diff
@@ -30,12 +32,11 @@ export function useAppLayout(
       containerSize = '2xl';
     } else if (containerWidth > 1024) {
       containerSize = 'xl';
-    } else if (containerWidth > 875) {
+    } else if (containerWidth > SIDEBYSIDE_THRESHOLD) {
       containerSize = 'lg';
-    } else if (containerWidth > 768) {
+    } else if (containerWidth >= SIDEBYSIDE_THRESHOLD - TOOLBAR_WIDTH) {
```

Comment:
These need to be tuned, but this update ensures they're "in sync" with the side-by-side mode, at least.